Symfony/CS/Tests/FixerFactoryTest.php

```diff
@@ -279,4 +279,24 @@ function ($name) {
 
         return $cases;
     }
+
+    public function testHasRule()
```

Comment:
`willReturn` will return `PHPUnit_Framework_MockObject_Builder_InvocationMocker`, and this is what you got under `$f1`
What you  need undef `$f1` is result of `getMock`, which would be `PHPUnit_Framework_MockObject_MockObject`